core/domain/classifier_domain.py

```diff
@@ -69,16 +69,14 @@ class ClassifierTrainingJob(python_utils.OBJECT):
                     'answers': ['a2', 'a3']
                 }
             ]
-        classifier_data: dict. The actual classifier model used for
-            classification purpose.
-        data_schema_version: int. Schema version of the data used by the
-            classifier. This depends on the algorithm ID.
+        algorithm_version: int. Schema version of the classsifier model to be
```

Comment:
"classifier" is misspelled. I've seen this a few times, maybe do a quick grep to catch other cases if there are any.

Reply:
Done